python/tests/ingestion/test_merge_utils.py

```diff
@@ -41,6 +40,19 @@
     ['100', '10101', 'BLACK_NH', 100, 50.0],
 ]
 
+_pop_data_time_series = [
+    ['time_period', 'state_fips', 'race_category_id', 'population', 'population_pct'],
+    ['2019', '01', 'BLACK_NH', 200, 50.0],
+    ['2019', '01', 'WHITE_NH', 200, 50.0],
+    ['2019', '02', 'BLACK_NH', 200, 50.0],
+    ['2019', '100', 'BLACK_NH', 200, 50.0],
+    [ACS_LATEST_YEAR, '01', 'BLACK_NH', 100, 25.0],
```

Comment:
Ya, I wasn't sure about this. I figured this might be better because we DO expect to change the `ACS_LATEST_YEAR` once a year, so I thought it'd be nice to just update in the datasource and then everything would pass. 

Would you rather have the tests fail when we annually update the year and we just manually remember to update the tests to match the new latest year?  I'm open to whatever you think is better practice

Reply:
I just swapped it for `2021` I was kind of thinking about it backwards. We don't have to update the tests this way, since even when the real data gives us new years the test data will still have 2021 as its latest". I added a few comments on the expected data json to explain what we should expect for each chunk